set.mm

```diff
@@ -613998,6 +614031,150 @@ number axioms (add ~ ax-10 , ~ ax-11 , ~ ax-13 , ~ ax-nul , and remove
       JZMHZGHABIJZMHZGHCMHZGHADNUAUCGTUBMSBAIEOPPUCUDGUBCMFPPQ $.
   $}
 
+  ${
+    carrydec.a $e |- A e. NN0 $.
+    carrydec.b $e |- B e. NN0 $.
+    carrydec.c $e |- C e. NN0 $.
+    carrydec.1 $e |- ( A + B ) = D $.
+    $( Carry a digit, removing a decimal constructor.  (Contributed by Steven
+       Nguyen, 10-Dec-2022.) $)
+    carrydec $p |- ; A ; B C = ; D C $=
```

Comment:
This and the following: probably the deduction versions would be more usable, i.e.,
```
$e |- ( ph -> A e. NN0 ) $.
...
$p |- ( ph -> ; A ; B C = ; D C ) $=
```

Reply:
By the way: I'm not sure that `; A ; B C = ; D C` is sensical, and even if that result is used later, I do not know if we want this kind of intermediate lemma in set.mm.  I haven't looked at that part of set.mm in detail, so maybe @digama0 can chime in.

Reply:
Although allowed by definition, I also would like to avoid such constructs (; should be used only as prefix). I did not find any such construct in set.mm yet.